src/ontology/cl-edit.owl

```diff
@@ -9500,8 +9500,12 @@ SubClassOf(obo:CL_0000705 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000097
 # Class: obo:CL_0000706 (choroid plexus epithelial cell)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:9550134") obo:IAO_0000115 obo:CL_0000706 "Specialized ependymal cell that produces the cerebrospinal fluid from the blood and secretes it into the lumen of the brain and spinal chord.")
+AnnotationAssertion(obo:RO_0002175 obo:CL_0000706 obo:NCBITaxon_9606)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_0000706 "2023-08-04T08:48:30Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2022.10.12.511898") oboInOwl:hasBroadSynonym obo:CL_0000706 "choroid plexus cell")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000706 "FMA:70549")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000706 "epithelial cell of choroid plexus")
+AnnotationAssertion(oboInOwl:inSubset obo:CL_0000706 <http://purl.obolibrary.org/obo/uberon/core#human_reference_atlas>)
```

Comment:
Please, remove the human_reference_atlas subset here. All tags should be in the `hra_subset.owl` which is automatically updated from the ASCT+B tables. The change to use the new term should be made there, and then it will be changed here when updating the imports.

Reply:
Ok, I will track down the term then to ask Ellen to update the term. Thank you Anita!

Reply:
Thanks, @anitacaron and @aleixpuigb.
human_reference_atlas subset annotation removed.
Kindly advise if anything else is needed prior to merge.